index.html

```diff
@@ -10,11 +10,13 @@
 
 $(document).ready(function(){
 
-    $('#button').click(function(e) {  
-        var inputvalue = $("#input").val();
-        var inputvalue2 = $("#input2").val();
-        window.location.replace(" http://www.paypal.me/"+inputvalue+"/"+inputvalue2);
+    $('#button').click(function(e) {
+      var username = $("#input").val();
+      var amount = $("#input2").val();
```

Comment:
You could do a `username = username.replace(/^(https?:\/\/)?paypal.me\//, '') on the link to get the `username` part of a full paypal.me link. (I think, not sure if my regular expression is correct.